namespaces/ovirt.yml

```diff
@@ -0,0 +1,27 @@
+
+namespace:
+  name: ovirt
+  type: group
+  description: >
+    Namespace for ovirt metadata.
+  fields:
+  - name: entity
+    type: string
+    description: >
+      The type of the data source, hosts, vms, engine etc.
+
+  - name: host_id
+    type: string
+    description: >
+      The oVirt host UUID.
+
+  - name: engine
+    type: group
```

Comment:
Since I'm thinking that we might add other engine metadata in the future.
What do you suggest?